package.yaml

```diff
@@ -14,7 +14,7 @@ synopsis: A non-empty comonadic list zipper
 description: A non-empty comonadic list zipper
 
 dependencies:
-  - base < 5
+  - base
```

Comment:
I think Hackage rejects and upload if you don't bound `base`. Does this project use `stack-upload-action`? If so, at what `pvp-bounds`?

Reply:
Updated `release.yml` to use the new release style.